packages/babel-helper-create-class-features-plugin/src/fields.ts

```diff
@@ -994,6 +1042,12 @@ function isNameOrLength({ key, computed }: t.ClassProperty) {
   return false;
 }
 
+function inheritPropComments<T extends t.Node>(node: T, prop: PropPath) {
+  t.inheritLeadingComments(node, prop.node);
+  t.inheritInnerComments(node, prop.node);
```

Comment:
Unlike `t.inheritsComments`, the `inheritPropComments` only inherits leading and inner comments. For every class member except the last one, its trailing comments must be the leading comments of the next sibling. The current approach favours the JSDoc style leading comments, and will lose the trailing comments of the last member. I think it does not worth the extra efforts to recover the last trailing comments. We can fix that in the future.

Reply:
Could you add the "For every class member [...] the trailing comments of the last member" part as a comment in the code? :)